lms/envs/common.py

```diff
@@ -3021,3 +3021,20 @@
 ############## Settings for LMS Context Sensitive Help ##############
 
 DOC_LINK_BASE_URL = None
+
+########################## Course Discovery #######################
+LANGUAGE_MAP = {'terms': {lang: display for lang, display in ALL_LANGUAGES}, 'name': 'Language'}
```

Comment:
I'm okay adding the pass-through to `ENV_TOKENS` above, but I'm not sure about moving this particular block from `devstack.py` to `common.py`.  I think in general we want to avoid having configuration values in `common.py` -- @fredsmith and @rlucioni are people I have worked with in the past in this particular area, so they may have more to say.